tests/test_fe_utils.py

```diff
@@ -32,3 +34,16 @@ def test_extract_delta_Us_from_U_knk():
     )
 
     np.testing.assert_almost_equal(expected_delta_Us, test_delta_Us)
+
+
+def test_image_molecules():
+    box_size = 10
+    box = np.eye(3) * box_size
+
+    np.random.seed(2022)
```

Comment:
Nit: Should random seed be set elsewhere in test file? (The test currently doesn't seem to use random numbers)

Request: Might also be good to test more cases, esp:
* offset by `(k * box_size) + eps` for some +ve and -ve integers `k`

Reply:
Woops, the random is an artifact of an earlier version of the test.

Will expand to test those cases

Reply:
One more test elaboration: could we assert here that pairwise distances are the same before and after imaging? (want to ensure `np.isclose(distance(x, box), distance(x_imaged, box))` for all tested cases `x`)

Reply:
I'd actually just check that the conformations are equivalent (or least separated by a translation) - we don't want this to have any rigid rotational component (which pairwise distances cannot detect).